src/core/curves.js

```diff
@@ -83,9 +103,9 @@ p5.prototype.bezier = function(x1, y1, x2, y2, x3, y3, x4, y4) {
  * </div>
  */
 p5.prototype.bezierDetail = function(d) {
```

Comment:
What's the reasoning for exposing bezierDetail as a public function?  Could you maybe provide an example where a user might want to call bezierDetail() ?